plugins/poll/config/locales/client.en.yml

```diff
@@ -22,7 +22,9 @@ en:
           title: "Results will be shown once <strong>closed</strong>."
         staff:
           title: "Results are only shown to <strong>staff</strong> members."
-
+        tabs:
+          votes: "Votes"
+          outcome: "Outcome"
```

Comment:
Are those translations specific to IRV?

Reply:
yes, I will remove from this PR, good catch.

Reply:
done: https://github.com/discourse/discourse/pull/27204/commits/7428538cfcfc459e92b983965cbf7a7158ba7ed8